### ISSUE TITLE
For public downloads make sure there's messaging that you don't need to enter user/pw. Also make script work w/o user/pw for public data. Capture who is trying to get the data using a prompt.

### ISSUE THREAD

**Issue by yuliujpl** [EDRN/labcas-ui#174]
## 🤔 Tell Us About the Feature

Replace this with a clear and concise description of the problem you're tying to solve; for example: I'm frustrated when … or I wish it were easier to …

## 🎇 What Solution You'd Like

Here, put a clear and concise description of what you want to happen.

## 〽️ Alternative Ideas

If you'd like, put in a description of any alternative solutions or features you've considered.

**Comment by hoodriverheather** [comment]
@yuliujpl It's fine to push this to happen after NIST sprint!

One other place we should think about capturing name and email on public download is if the file is smaller and is downloaded via browser.

**Comment by yuliujpl** [comment]
@hoodriverheather taking a further step back, should we just capture download analytics comprehensively then? Whoever downloads what and stats associated with them? That way the download tracking is centralized but it's just different ways to get user data either via login or via per-download name/email form they have to fill out?

**Comment by hoodriverheather** [comment]
@yuliujpl and @nutjob4life Are we capturing download analytics for LabCAS?

**Comment by nutjob4life** [comment]
@hoodriverheather the backend creates a log file in `/usr/local/labcas/backend/home/download.log` with one line per downloaded file. The format is

    timestamp;user-DN;file

Example from the file on `edrn-labcas`:
```
2025-08-22T15:23:22.882-07;uid=jani66fuju,dc=edrn,dc=jpl,dc=nasa,dc=gov;Pre-diagnostic_PDAC_Images/UPMC/UPMC_00470/UPMC_Pre_Dx/CT1.2.840.113619.2.55.3.2047482071.5582.1175161253.294.54.dcm
2025-08-22T15:23:24.933-07;uid=jani66fuju,dc=edrn,dc=jpl,dc=nasa,dc=gov;Pre-diagnostic_PDAC_Images/UPMC/UPMC_00470/UPMC_Pre_Dx/CT1.2.840.113619.2.55.3.2047482071.5582.1175161253.294.55.dcm
2025-08-22T15:23:26.978-07;uid=jani66fuju,dc=edrn,dc=jpl,dc=nasa,dc=gov;Pre-diagnostic_PDAC_Images/UPMC/UPMC_00470/UPMC_Pre_Dx/CT1.2.840.113619.2.55.3.2047482071.5582.1175161253.294.102.dcm
2025-08-25T13:23:40.986-07;uid=emily,dc=edrn,dc=jpl,dc=nasa,dc=gov;Automated_Quantitative_Measures_of_Breast_Density_Data/C0250/MASK/C0250_MASK_PRO_LCC.dcm
```

**Comment by hoodriverheather** [comment]
@nutjob4life Is there a size of the download? just curious. i can ask Dan what he would want for stats.

**Comment by nutjob4life** [comment]
@hoodriverheather no, there isn't any size; see [the example](https://github.com/EDRN/labcas-ui/issues/174#issuecomment-3225480235).

**Comment by hoodriverheather** [comment]
@nutjob4life is it difficult to add additional information to this log?

**Comment by nutjob4life** [comment]
@hoodriverheather depends on the scope of the information.

For example, adding a file size is easy.
